otter/metrics.py

```diff
@@ -41,6 +41,23 @@
 metrics_log = otter_log.bind(system='otter.metrics')
 
 
+QUERY_GROUPS_OF_TENANTS = (
+    'SELECT '
+    '"tenantId", "groupId", desired, active, pending, created_at, status '
+    'FROM scaling_group WHERE "tenantId" IN ({tids})')
+
+
+@defer.inlineCallbacks
+def get_specific_scaling_groups(client, tenant_ids):
+    tids = ', '.join("'{}'".format(tid) for tid in tenant_ids)
+    query = QUERY_GROUPS_OF_TENANTS.format(tids=tids)
+    results = yield client.execute(query, {}, ConsistencyLevel.ONE)
+    defer.returnValue(r for r in results
+                      if r.get('created_at') is not None and
+                      r.get('desired') is not None and
+                      r.get('status') != 'DISABLED')
```

Comment:
You might want add a check for "ERROR" too unless https://github.com/rackerlabs/otter/issues/1304 is implemented